src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -631,6 +633,13 @@ public static SaplingGrowTreeEvent blockGrowFeature(LevelAccessor level, RandomS
         return event;
     }
 
+    public static AlterGroundEvent alterGround(LevelSimulatedReader level, RandomSource random, BlockPos pos, BlockState altered)
```

Comment:
With the message on the patch, it could probably be reduced to one line if this returned a blockstate instead.